docs/src/idiomatic/assignments.md

```diff
@@ -0,0 +1,129 @@
+# Idiom 1: Update state variables with assignments
+For each TLA+ variable `x` defined in a specification, Apalache requires that every possible  transition defines an assignment to `x'`.
```

Comment:
I think it would be helpful to also capture the intuitive concept here, not just the operational requirement. So maybe with an additional sentence like:

```suggestion
For each TLA+ variable `x` defined in a specification, Apalache requires that every possible  transition defines an assignment to `x'`. I.e., every possible transition must specify the next value after the transition for each TLA+ variable.
```